app/assets/stylesheets/application.css

```diff
@@ -1176,7 +1171,7 @@ a.remove-from-my-own:hover {
 .unreads-list {
   display: flex;
   flex-wrap: wrap;
-  align-items: center;
+  align-items: baseline;
```

Comment:
試しにtopにしてみると

```suggestion
  align-items: top;
```

こんな感じの上揃えになって納まりがよさそうな感触なんですが @sugiwe さん的にはどうでしょう？ :smiley:

![Unreads_-_rururu_-_Vivaldi](https://github.com/user-attachments/assets/c4c05d74-1723-459a-98ba-69bff14b9d7e)

Reply:
こちらありがとうございます！
この部分ちょっと迷っていまして、先ほど検討していた時は、`top`だと下の余白に比べて上が詰まって見えるのが個人的に気になってしまい、偶然ではあるのですが`baseline`が程よい頃合いになるから良いかなぁと思っていた、という経緯でした。
なのですが、確かに`top`のが上が揃って気持ちが良い感じがしますし、結局`baseline`は中途半端な気がしてきたので、`top`で良いなと改めて感じました！あとPodcastの場合（再生バーがある場合）などに`baseline`はあまり収まりが良くない気もしてきました😂
なので、topにしてプッシュしますね✨